chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -293,29 +293,44 @@ out:
 			switch item := ntfn.(type) {
 			case chain.BlockConnected:
 				if item.Height != b.bestBlock.Height+1 {
-					chainntnfs.Log.Warnf("Received blocks out of order: "+
-						"current height=%d, new height=%d",
-						bestHeight, item.Height)
-					continue
-				}
-
-				rawBlock, err := b.chainConn.GetBlock(&item.Hash)
-				if err != nil {
-					chainntnfs.Log.Errorf("Unable to get block: %v", err)
-					continue
-				}
+					// Handle the case where the notifier
```

Comment:
what if `item.Height < b.bestBlock.Height+1`, maybe we should just log and continue in that case?

Reply:
hm good idea, fixed

Reply:
actually, there is technically an edge case where we miss multiple `disconnected block` notifications then receive a `connected block` notification with a height less than ours